NEWS.md

```diff
@@ -7,8 +7,7 @@ Changes in version 0.14
 
 **NOT RELEASED YET; STILL UNDER DEVELOPMENT.**
 
-* No changes recorded.
-
```

Comment:
One last thing: you removed two lines but added one. Note that all other subsections are separate from each other by two blank lines, not one.

Also, the proper spelling is C++11, without the dash.